universal-application-tool-0.0.1/app/views/style/ApplicantStyles.java

```diff
@@ -77,6 +78,8 @@
       StyleUtils.joinStyles(BUTTON_BASE, Styles.TEXT_SM, Styles.MX_AUTO);
   public static final String BUTTON_BLOCK_NEXT =
       StyleUtils.joinStyles(BUTTON_BASE, Styles.TEXT_BASE, Styles.MR_0, Styles.ML_AUTO);
+  public static final String BUTTON_REVIEW =
```

Comment:
(no action) I may play with the style on this a bit later. Looks reasonable for now though!